tools/gulp/tasks/development.ts

```diff
@@ -28,3 +37,10 @@ task(':serve:devapp', serverTask(outDir, true));
 task('serve:devapp', ['build:devapp'], sequenceTask(
   [':serve:devapp', 'material:watch', ':watch:devapp']
 ));
+
+/** Task that copies all vendors into the demo-app package. Allows hosting the app on firebase. */
+task('prepare-deploy:devapp', ['build:devapp'], () => {
```

Comment:
@jelbourn Not sure, didn't try it with the Firebase-Tools API yet. Will try it later.

With the `goOffline()` stuff I meant that you manually need to "close" the firebase connection, because otherwise the Gulp task will run forever.

@mmalerba That would be an option, but sounds IMO a bit weird to parse arguments in an gulp task.

The advantage of having a task like `prepare-deploy` is, that we might want to make the demo-app "deploy-able", but don't want to publish it immediately. e.g when having the PR previews bot.

Let me see if I need to specify a firebase project.